packages/common/src/pipe/transform.pipe.ts

```diff
@@ -19,21 +19,22 @@ export class TransformPipe implements DiscordPipeTransform {
   ): any {
     if (!metadata.metatype || !interaction || !interaction.isCommand()) return;
 
-    const { dtoInstance } = metadata.commandNode;
+    const { dtoInstance: originalInstance } = metadata.commandNode;
+    const newInstance = Object.assign({}, originalInstance);
```

Comment:
This `Object.assign({}, originalInstance)` cause the `ValidationPipe` to not work anymore :)

A fix in https://github.com/fjodor-rybakov/discord-nestjs/pull/616 was made to make this new logic work with ValidationPipe (aka cloning the object by keeping the originalInstance's class)